Panic on routing rules

Log: 
```
panic: runtime error: index out of range

goroutine 16 [running]:
encoding/binary.bigEndian.Uint64(...)
        /snap/go/4301/src/encoding/binary/binary.go:124
github.com/skycoin/skywire/pkg/routing.Rule.KeepAlive(...)
        /home/evanlinjin/dev/skycoin/skywire/pkg/routing/rule.go:47
github.com/skycoin/skywire/pkg/router.(*managedRoutingTable).ruleIsTimedOut(0xc000252ae0, 0x2, 0x0, 0x0, 0x0, 0x0)
        /home/evanlinjin/dev/skycoin/skywire/pkg/router/managed_routing_table.go:91 +0x28a
github.com/skycoin/skywire/pkg/router.(*managedRoutingTable).Rule(0xc000252ae0, 0xc000000002, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/evanlinjin/dev/skycoin/skywire/pkg/router/managed_routing_table.go:54 +0x11c
github.com/skycoin/skywire/pkg/router.(*routeManager).GetRule(0xc00025e9c0, 0x2, 0xd33278, 0xc000041400, 0xc0000f3ef0, 0x42ecc6, 0xc0000a7860)
        /home/evanlinjin/dev/skycoin/skywire/pkg/router/route_manager.go:174 +0x77
github.com/skycoin/skywire/pkg/router.(*Router).handlePacket(0xc000268230, 0xe0d640, 0xc0000c8010, 0xc000453640, 0x10, 0x10, 0x0, 0x0)
        /home/evanlinjin/dev/skycoin/skywire/pkg/router/router.go:137 +0x1b9
github.com/skycoin/skywire/pkg/router.(*Router).Serve.func1(0xc000268230, 0xe0d640, 0xc0000c8010)
        /home/evanlinjin/dev/skycoin/skywire/pkg/router/router.go:116 +0x107
created by github.com/skycoin/skywire/pkg/router.(*Router).Serve
        /home/evanlinjin/dev/skycoin/skywire/pkg/router/router.go:110 +0x109
```

--- a/pkg/routing/boltdb_routing_table.go
+++ b/pkg/routing/boltdb_routing_table.go
@@ -76,7 +76,9 @@ func (rt *boltDBRoutingTable) Rule(routeID RouteID) (Rule, error) {
 		rule = b.Get(binaryID(routeID))
 		return nil
 	})
-
+	if rule == nil {
+		return nil, fmt.Errorf("rule of routeID '%v' does not exist", routeID)
+	}
 	return rule, err
 }
 

